base/applications/rapps/include/appdb.h

```diff
@@ -18,6 +18,8 @@ class CAppDB
   public:
     CAppDB(const CStringW &path);
 
+    BOOL bAppwizMode;
```

Comment:
How is this at all related to the application db?

Reply:
Stas'M:

>   How is this at all related to the application db?

I didn&#x27;t find a namespace to share a variable between unattended.cpp and gui.cpp -- it&#x27;s not related to the appdb, but it&#x27;s the simplest way to add it

Another options are:
● Add a global variable somewhere
● Add field to Settings.xxx global variable

In the first case I&#x27;m not fan of adding new global vars :) Second case would have required some additional logic and hacks.

Reply:
We can use `GlobalAtom` or `LocalAtom`... Not recommended though...

Reply:
I'd propose to put this in the `SETTINGS_INFO` struct.
This can later (in another PR) be refactored to read all settings from the command line (or a file) as override on the registry settings.
e.g. First read the registry, then read from the command line (as override).

Reply:
@learn-more This should now be addressed.